### ISSUE TITLE
The filter apps are creating zombies.

### ISSUE THREAD

**Issue by ppywlkiqletw** [OpenPrinting/cups-filters#430]
In the program ppd/ppd-collection.cxx, whenever you call PipeCommand, you should also call wait() or waitpid() so the zombies can be put to rest.

```
    if ((fp = PipeCommand(&pid, realname, argv, 0)) != NULL)
    {
      if ((fd = cupsTempFd(tempname, sizeof(tempname))) < 0)
      {
        if (log) log(ld, FILTER_LOGLEVEL_ERROR,
                     "libppd: [PPD Collections] Unable to copy PPD to temp "
                     "file: %s",
                     strerror(errno));
        return (NULL);
      }
      while ((bytes = cupsFileRead(fp, buffer, sizeof(buffer))) > 0)
        bytes = write(fd, buffer, bytes);
      cupsFileClose(fp);
      close(fd);
      fp = cupsFileOpen(tempname, "r");
      unlink(tempname);
>>>>>> call wait() here to put the zombie to rest.
    }
    else
    {
      if (log) log(ld, FILTER_LOGLEVEL_WARN,
                   "libppd: [PPD Collections] Unable to execute \"%s\": %s",
                   tempname, strerror(errno));
      return(NULL);
    }

```  

Same for the other place where PipeCommand is called.

**Comment by tillkamppeter** [comment]
Fixed in commit 3260d9a1. Please test

Thank you for the bug report.

**Comment by ppywlkiqletw** [comment]
> Fixed in commit [3260d9a](https://github.com/OpenPrinting/cups-filters/commit/3260d9a19e902ccd0c88bc3dec8c7d98ce1375c9). Please test
> 
> Thank you for the bug report.

Perfect.   No zombies when running "ps axf".
